engine/tester/config_parse.c

```diff
@@ -2136,6 +2265,8 @@ alloc_and_get_run_item(xmlNodePtr node, tester_cfg *cfg, unsigned int opts,
         }
         node = xmlNodeNext(node);
     }
+    if (p->type == RUN_ITEM_SCRIPT && p->tmpl != NULL)
```

Comment:
I'm not sure that it is the right place to inherit template args, but type check is inappropriate here. Just p->tmpl != NULL

Reply:
I moved it just after get_script call. And for session adding of argument it called in get_session() function.